sky/client/sdk.py

```diff
@@ -1944,6 +1944,7 @@ def stream_and_get(request_id: None = None,
 @usage_lib.entrypoint
 @server_common.check_server_healthy_or_start
 @annotations.client_api
+@rest.retry_transient_errors()
```

Comment:
importing functions from the `server` module in the client is a bit weird. We can do a refactor in the future.